src/site/content/en/blog/bbc-hsts/index.md

```diff
@@ -0,0 +1,74 @@
+subhead: |
+  The BBC is rolling out HSTS for their website to improve security and performance. Find out what it means, and how HSTS can help you.
+authors:
+  - jlwagner
+date: 2022-07-25
+description: |
+  The BBC is rolling out HSTS for their website to improve security and performance. Find out what it means, and how HSTS can help you.
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/BbUZ7XxZ991yp4lpD5Tr.png
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ew6tI1HdAi2g3vxQwdfM.png
+alt: The BBC logo set on a black background.
+tags:
+  - blog
+  - performance
+  - security
+---
+
+HTTPS adoption has been steadily increasing in recent years. Per the HTTP Archive's 2021 Web Almanac, [around 91% of all requests for both desktop and mobile were served over HTTPS](https://almanac.httparchive.org/en/2021/security#transport-security). HTTPS isn't just here to stay, it's a necessary prerequisite to use features such as [Service Worker](https://developer.mozilla.org/docs/Web/API/Service_Worker_API) and modern protocols such as [HTTP/2](/performance-http2/) and [HTTP/3](https://www.cloudflare.com/learning/performance/what-is-http3/).
+
+Recently Neil Craig&mdash;a lead technical Architect at the BBC&mdash;tweeted that [HTTP Strict Transport Security (HSTS)](https://developer.mozilla.org/docs/Web/HTTP/Headers/Strict-Transport-Security) is being slowly rolled out for [bbc.com](https://www.bbc.com/). Let's find out what that means for the BBC, and what it could mean for you.
+
+## The problem
+
+Web servers these days tend to listen for requests on ports 80 and 443. Port 80 is for insecure HTTP requests, whereas 443 is for secure HTTPS. This can create a problem, because when you enter an address into your address bar without the https:// protocol prefix&mdash;like most users tend to do&mdash;some browsers will direct traffic to the insecure HTTP version of a site.
+
+A common way to ensure users don't access an unsecured version of a website is to place an HTTP-to-HTTPS redirect for all requests. This certainly works, but it kicks off the following chain of events:
+
+1. The server receives a request via HTTP.
+2. The server issues a redirect to go to the HTTPS equivalent of the requested resource.
+3. The server via HTTPS must negotiate a secure connection with the browser.
+4. The content loads as usual.
+
+The problem here is that the 301 redirect delays rendering of content, forcing the user to wait longer, and unless redirects are properly set up, you can still end up sending users to an insecure version of a web site.
+
+## Enter HSTS
+
+{% BrowserCompat 'HTTP.Headers.Strict-Transport-Security' %}
+
+HSTS is dictated by the `Strict-Transport-Security` HTTP response header for HTTPS requests. When set, return visits to a website will trigger a special redirect known as a "307 Internal Redirect", which is when the browser handles the redirect logic. These types of redirect are extremely fast, and add no noticeable latency during an HTTP-to-HTTPS hop.
```

Comment:
```suggestion
HSTS is dictated by the `Strict-Transport-Security` HTTP response header for HTTPS requests. When set, return visits to a website will trigger a special redirect known as a "307 Internal Redirect", which is when the browser handles the redirect logic, rather than second it to the server. This prevents the request being intercepted, since it never leaves the browser, so is more secure. As and added bonus, as there is no network request needed to get the redirect, these types of redirect are extremely fast, and so prevent noticeable latency during an HTTP-to-HTTPS hop.
```